Blizzard.IO.RabbitMQ/RabbitPropertiesConverter.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Blizzard.IO.Core;
+using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using System;
+using System.Collections.Generic;
+
+namespace Blizzard.IO.RabbitMQ
+{
+    public class RabbitPropertiesConverter :
+        IConverter<RabbitMessageProperties, MessageProperties>,
+        IConverter<MessageProperties, RabbitMessageProperties>
+    {
+        public MessageProperties Convert(RabbitMessageProperties obj)
```

Comment:
```suggestion
        public MessageProperties Convert(RabbitMessageProperties rabbitMessageProperties)
```
It looks more good then with obj